### PR TITLE
Initial attempt at adding usage docs

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

First draft of basic input documentation, as I guess we are getting the point people could start running the pipeline now.

TODO: Add to usage some basic explanations of the different preprocessing options. 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,20 +32,20 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
    - Run merging
-3. Performs taxonomic profiling a choice of:
-   - Kraken2
-   - MetaPhlAn3
-   - MALT
-   - DIAMOND
-   - Centrifuge
-   - Kaiju
-   - mOTUs
+3. Performs taxonomic profiling via a choice of any or all of:
```
Comment (Midnighter) [inline README.md:39]:
```suggestion
3. Performs taxonomic profiling using one or more of:
```

File: docs/usage.md [hunk 2]
```diff
@@ -8,56 +8,90 @@
...
 
 ```console
---input '[path to samplesheet file]'
+--input '[path to samplesheet file]' --databases '[path to database sheet file]'
 ```
 
 ### Multiple runs of the same sample
 
-The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
+The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will processed reads before performing profiling. Below is an example for the same sample sequenced across 3 lanes:
```
Comment (Midnighter) [inline docs/usage.md:23]:
```suggestion
The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will process reads before performing profiling. Below is an example for the same sample sequenced across 3 lanes:
```

File: docs/usage.md [hunk 3]
```diff
@@ -8,56 +8,90 @@
...
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
+sample,run_accession,instrument_platform,fastq_1,fastq_2,fasta
+2612,run1,ILLUMINA,2612_run1_R1.fq.gz,,
+2612,run2,ILLUMINA,2612_run2_R1.fq.gz,,
+2612,run3,ILLUMINA,2612_run3_R1.fq.gz,2612_run3_R2.fq.gz,
+
 ```
 
+> ⚠️ Runs of the sample sample sequenced on Illumina platforms with a combination of single and paired-end data will **not** be run-wise concatenated, unless pair-merging is specified. In the example above, `run3` will be profiled independently of `run1` and `run2` if pairs not merged.
```
Comment (Midnighter) [inline docs/usage.md:33]:
```suggestion
> ⚠️ Runs of the same sample sequenced on Illumina platforms with a combination of single and paired-end data will **not** be run-wise concatenated, unless pair-merging is specified. In the example above, `run3` will be profiled independently of `run1` and `run2` if pairs are not merged.
```

File: docs/usage.md [hunk 4]
```diff
@@ -8,56 +8,90 @@
...
+| `fastq_2`             | Path or URL to Illumina R2 sequencing reads in FASTQ format. GZipped compressed files accepted. Can be left empty if single end data. Cannot be combined with `fasta`.                                   |
+| `fasta`               | Path or URL to long-reads or contigs in FASTA format. GZipped compressed files accepted. Can be left empty if data in FASTA is specifed. Cannot be combined with `fastq_1` or `fastq_2`.                 |
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+### Full database sheet
+
+nf-core/taxprofiler supports multiple databases being profiled in parallel for each tool. These databases, and specific parameters for each, can be specified in a 4 column comma-separated sheet.
+
+> ⚠️ nf-core/taxprofiler does not provide any databases by default, nor currently generates them for you. This must be performed manually by the user.
```
Comment (Midnighter) [inline docs/usage.md:64]: In future, we should point to build pipelines here.

```suggestion
> ⚠️ nf-core/taxprofiler does not provide any databases by default, nor does it currently generate them for you. This must be performed manually by the user.
```

**Review by Midnighter — APPROVED** [review]
Thanks for adding some documentation. I noticed that you use profiling pretty exclusively. Maybe we need to define how we use certain terms, i.e., do we treat profiling, classification, and read mapping as synonyms or not?

**Comment by jfy133** [comment]
> Thanks for adding some documentation. I noticed that you use profiling pretty exclusively. Maybe we need to define how we use certain terms, i.e., do we treat profiling, classification, and read mapping as synonyms or not?

Yes, it's someting I was thinking about while writing so decided to stick on one for now. Maybe we should poll this on slack?

**Pull request merged**